### PR TITLE
Adds feature to jump to Error/warning from the BuildOutput treeview +…

### PR DESCRIPTION
**Pull request by netonjm**
Adds feature to jump to Error/warning from the BuildOutput treeview + test

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputWidget.cs [hunk 1]
```diff
@@ -191,6 +205,25 @@ void Initialize ()
 			scrolledWindow = new ScrollView { BorderVisible = false };
 			scrolledWindow.Content = treeView;
 
+			treeView.ButtonPressed += delegate (object sender, ButtonEventArgs e) {
```
Comment (jtorres) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputWidget.cs]: could you use a handler and unsubscribe it?, this class implements IDisposable

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputViewContent.cs [hunk 2]
```diff
@@ -46,13 +47,15 @@ public BuildOutputViewContent (FilePath filename)
 			this.ContentName = filename;
 			control = new BuildOutputWidget (filename);
 			control.FileSaved += FileNameChanged;
+			control.TaskSelected += (s, e) => TaskSelected?.Invoke (s, e);
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputViewContent.cs]: Unsubcribe events please.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputNode.cs [hunk 3]
```diff
@@ -111,6 +111,27 @@ public void AddParameter (string message, string fullMessage)
 			});
 		}
 
+		public BuildOutputNode InverseSearchFirstNode (BuildOutputNodeType type, string search = null)
+		{
+			if (type == NodeType) {
+				if (search == Message || search == null) {
+					return this;
+				} else {
```
Comment (rodrmoya) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputNode.cs]: if search is null or empty, it should return no items, right?

Comment (netonjm) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputNode.cs]: if search is null, it means we are only searching the first type coincidence

**Pull request merged**